bundle/edu.gemini.spModel.io/src/main/scala/edu/gemini/spModel/io/impl/migration/to2015B/BrightnessParser.scala

```diff
@@ -19,35 +19,49 @@ object BrightnessParser extends RegexParsers {
     """\s*[~=,]?\s*""".r ^^^ { () }
 
   private val at: Parser[Unit] =
-    """\s*(nucleus at|at|@|/|:)?\s*""".r ^^^ { () }
+    """\s*(nucleus at|at|in|@|/|:)?\s*""".r ^^^ { () }
 
-  private val num: Parser[Double] =
-    """-?\d+(\.\d*)?""".r <~ """\s*(mag)?""".r ^^ (_.toDouble)
+  private val num: Parser[BigDecimal] =
+    """-?\d+(\.\d*)?""".r <~ """\s*(mag)?""".r ^^ (s => BigDecimal(new java.math.BigDecimal(s))) // BigDecimal.exact(s)
```

Comment:
I felt a bit sheepish about hauling in `BigDecimal` but I wanted a more accurate way to convert, say, 200 mJy to 0.2 Jy.  Normal floating point math left values with tiny errors that break the test cases at any rate.